app/models/ext_management_system.rb

```diff
@@ -36,6 +36,7 @@ def self.supported_types_and_descriptions_hash
   has_many :ems_events,     -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "ems_id"
   has_many :policy_events,  -> { order "timestamp" }, :class_name => "PolicyEvent", :foreign_key => "ems_id"
 
+  has_many :event_blacklists, :foreign_key => "ems_id", :dependent => :destroy
```

Comment:
Note that the classname feels weird in the same way

`EventBlacklist.count`  sounds like you are counting lists
So maybe the class should be `EventBlacklisting` as well